contracts/src/protocol/OmniAVS.sol

```diff
@@ -197,19 +203,69 @@ contract OmniAVS is IOmniAVS, IOmniAVSAdmin, IServiceManager, OwnableUpgradeable
         xcallGasLimitPerValidator = perValidator;
     }
 
+    /// @inheritdoc IOmniAVSAdmin
+    function addToAllowlist(address operator) external onlyOwner {
+        require(operator != address(0), "OmniAVS: zero address");
+
+        for (uint256 i = 0; i < _allowlist.length; i++) {
+            require(_allowlist[i] != operator, "OmniAVS: already in allowlist");
+        }
+
+        _allowlist.push(operator);
+    }
+
+    /// @inheritdoc IOmniAVSAdmin
+    function removeFromAllowlist(address operator) external onlyOwner {
+        for (uint256 i = 0; i < _allowlist.length; i++) {
+            if (_allowlist[i] == operator) {
+                _allowlist[i] = _allowlist[_allowlist.length - 1];
+                _allowlist.pop();
+                return;
+            }
+        }
+        revert("OmniAVS: not in allowlist");
```

Comment:
We revert here, but don't revert in `removeOperator`, and check if the operator exists first in another loop. I think we should follow a single pattern - which one do you prefer?